### PR TITLE
[teams] use owner's GitLab account by default

### LINKED ISSUE #5331: Default to personal account when importing a project from GitLab
### Problem to solve

Following up from https://github.com/gitpod-io/gitpod/pull/5120#discussion_r690469441, when trying to import a project from GitLab,  the pre-selected account is a specific but maybe random group.

### Proposal

Default to personal account when importing a project from GitLab. 🦊

### PR DESCRIPTION
**Pull request by AlexTugarev**
## Description
When creating a new project from gitlab.com one's personal account should be preselected.

## Related Issue(s)

Fixes #5331

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **alextugarev** after the PR has been reviewed.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **alextugarev** after the PR has been reviewed.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)**
- **[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by AlexTugarev** [comment]
@gtsiolis, while implementing this change I realized that this somewhat contradicts the wanted behavior for GitHub accounts where we want to preselect the account which received the most recent update of the app's installation. For instance, if I recently added Gitpod's App to gitbhub.com/myawesomeaccont123, I want to see that one preselected. 

a) Should we explicitly diverge the provider behaviors? 
b) Or should we stick to a single rule? 

I've made a change for a) but it doesn't sound well enough, IMO.

**Comment by gtsiolis** [comment]
Looking at this now! 👀

**Comment by AlexTugarev** [comment]
@gtsiolis, please have a look

**Comment by gtsiolis** [comment]
Looking at this now! 👀

**Review by gtsiolis — APPROVED** [review]
This works great, @AlexTugarev! 🖖 

I could test this using GitLab, works as expected, and feels right. ✔️ 

Regarding the comment in https://github.com/gitpod-io/gitpod/pull/5900#issuecomment-929127040, I'd say let's go with the explicit diverge for the providers and improve later if needed after seeing this in action at large. 🚢 

Approving to unblock merging but holding in case this needs a closer look at the code.

/hold

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: aea62dc3645bc56700c99080f7d5403cb6f685a5</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5900#pullrequestreview-767746576" title="Approved">gtsiolis</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **alextugarev** after the PR has been reviewed.

Associated issue: *#5331*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [gtsiolis]
- **[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by AlexTugarev** [comment]
/hold cancel

Let's solve issues in follow ups

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5900#issuecomment-931405421" title="Approved">gtsiolis</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **alextugarev** after the PR has been reviewed.

Associated issue: *#5331*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [gtsiolis]
- **[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by gtsiolis** [comment]
Ah, @AlexTugarev we need someone from `server` component owners to also approve[[1](https://github.com/gitpod-io/gitpod/pull/5900#issuecomment-931405535)], right?

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5900#issuecomment-931957001" title="Approved">AlexTugarev</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/5900#issuecomment-931405421" title="Approved">gtsiolis</a>*

Associated issue: *#5331*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [AlexTugarev,gtsiolis]
- ~~[components/server/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/server/OWNERS)~~ [AlexTugarev]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
